### DISCUSSION TITLE
Hey shiftrauma works but....

### DISCUSSION THREAD

**Discussion by Militaria5678** [FakeFishGames/Barotrauma#14893] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

When i change into the monsters and use power everything is good but  when i turn into human again im not myself like im a white capitan and then im black medic for example

### Reproduction steps

1.make dedicated server lua 2. get any ball 3.turn into monester 4.turn back into human

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This seems like an issue in the mod, I'm afraid there's not much we can do to address the issue. You should probably report it to the creator of the mod instead.
